flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
this is disappointing, really.

Reply:
isn't it how it is supposed to be if we are only building for x86_64?

Reply:
it remain it is disappointing.

Reply:
ah now I get what you meant by that, yeah it is, so I am waiting for Tauri action to support producing aarch64 binaries.